api/helpers/game-states/moves/concede/execute.js

```diff
@@ -20,6 +20,11 @@ module.exports = {
       description: '0 or 1 for whether p0 or p1 is making the move',
       required: true,
     },
+    priorStates: {
+      type: 'ref',
+      description: "List of packed gameStateRows for this game's prior states",
+      required: true,
+    }
```

Comment:
Added `priorStates` to all `execute()` and `validate()` helpers so that they can accept this input given that the caller in `move.js` calls all of them with the same arguments and its needed for some of them. Little awkward, but at least it's consistent